src/metaswitch/clearwater/cluster_manager/main.py

```diff
@@ -54,6 +54,7 @@
   --log-level=LVL                Level to log at, 0-4 [default: 3]
   --log-directory=DIR            Directory to log to [default: ./]
   --pidfile=FILE                 Pidfile to write [default: ./cluster-manager.pid]
+  --cluster-manager-enabled=BOOL Whether the cluster manager should start any threads [default: Yes]
```

Comment:
Given that it's not a true/false bool, might be worth stating that the options are `[Y/N]`, with a default of `Y`